### ISSUE TITLE
Make nodes composable

### ISSUE THREAD

**Issue by rjoomen** [tesseract-robotics/tesseract_ros2#52]
Make nodes composable as components.

**Comment by Levi-Armstrong** [comment]
@rjoomen Has this been addressed?

**Comment by rjoomen** [comment]
No. And I've not started with this yet on my own work either, so it might be a while before I even get around to looking into this. Shall we close the issue?
